index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    
+    <header>
+        <nav class="menu_icon">
+                <nav>
+                    <a href="#menu" class="box-shadow-menu"></a>
+                </nav>
+        </nav>
+    </header>   
+    
+    <section class="main_section">
+        <section class="home_section">
+            
+            <h1 class="upper_word"><br>Hey there.<br> I’m Ishimwe.</h1>
+            <h1 class="down_word">I’m a software developer</h1>
```

Comment:
- [ ] [OPTIONAL] It is not recommended to use more than one `h1` tag on a landing page. The `h1` tag should be used to indicate the main heading or title of the page and should be unique to that page. Having multiple `h1` tags can confuse search engines and users about the main topic of the page and can negatively impact the page's ranking in search results. 
- [ ] So I'd suggest you place the text "I’m a software developer" in an `h2` tag to follow the proper semantics. 😉

Reply:
Thanks